task11/12.sql

```diff
@@ -0,0 +1,7 @@
+SELECT USER.USERNAME, IFNULL(REVIWS_AMOUNT.AMOUNT_REVIEWS, 0) AS AMOUNT_REVIEWS FROM discount.USER
```

Comment:
![image](https://user-images.githubusercontent.com/48795460/119506828-f63e4a00-bd76-11eb-96bb-952e8940ea0f.png)
https://github.com/marybash/WebProject/pull/12/files

Reply:
ok, but what is my rating for the task5 ,task8 and task11?